meta-flex-distro/recipes-devtools/perfetto/perfetto/0001-ftrace-Avoid-crashing-if-format-doesn-t-match-expect.patch

```diff
@@ -0,0 +1,42 @@
+From 989cecf48f5785a35925e3c818cb968344334ac3 Mon Sep 17 00:00:00 2001
+From: Daniele Di Proietto <ddiproietto@google.com>
+Date: Tue, 9 May 2023 16:10:03 +0000
+Subject: [PATCH 1/1] ftrace: Avoid crashing if format doesn't match
```

Comment:
Seems like perfetto uses FTRACE functionality from Linux kernel. Can you confirm if we need to enable this kernel configuration? Maybe by trying out some example related to function tracing and see if functions' entry and exit points are being displayed properly.

Reply:
https://gist.github.com/johnstultz-work/0ec4974e0929c4707bfd89c876ae4735#kernel-config